app/main/helpers/services.py

```diff
@@ -74,3 +74,9 @@ def get_next_section_name(content, current_section_id):
         return content.get_section(
             content.get_next_editable_section_id(current_section_id)
         ).name
+
+
+def get_containing_section(content, question):
```

Comment:
Do we still need this now that the URL includes the section_id? It seems we can do the same thing we do in the edit section endpoint